autoscheduler/frontend/src/components/SmallFastProgress.tsx

```diff
@@ -1,8 +1,18 @@
 import * as React from 'react';
 import { CircularProgress } from '@material-ui/core';
 
-const SmallFastProgress: React.FC = () => (
-  <CircularProgress size={24} style={{ animationDuration: '550ms' }} disableShrink />
-);
+interface SmallFastProgressProps {
```

Comment:
Since this is just going to be a POST/PUT request and the user won't be receiving any data from it (and as such nothing in the view will actually change), I don't think this needs a progress indicator at all & I think it's safe to assume that the request we do worked successfully. Worst case scenario we can show a snackbar in case of an error (although I don't think it's necessary)

Reply:
I disagree (at least about removing it entirely), since the most useful part of saving schedules is that after refreshing they'll still be there we want to make sure request to save actually goes through. Imagine that you click save on your perfect schedule, see "saved", refresh, and then it disappears when you refresh. Also because the overhead on the save/remove schedule API calls will probably be very small, I think it's acceptable to have the loading indicator